docs/docs/benchmarks-big-bench-hard.mdx

```diff
@@ -0,0 +1,98 @@
+---
+id: benchmarks-big-bench-hard
+title: BIG-Bench Hard
+sidebar_label: BIG-Bench Hard
+---
+
+The **BIG-Bench Hard (BBH)** benchmark comprises 23 challenging BIG-Bench tasks where prior language model evaluations have not outperformed the average human rater. BBH evaluates models using both few-shot and chain-of-thought (CoT) prompting techniques. For more details, you can [visit the BIG-Bench Hard GitHub page](https://github.com/suzgunmirac/BIG-Bench-Hard).
+
+## Arguments
+
+To evaluate your model using `BigBenchHard`, you'll have to provide the following arguments:
+
+- [Optional] `tasks`: a list of `BigBenchHardTask` enums, which specifies the subject areas for sentence completion evaluation. By default, this is set to all tasks. The list of `BigBenchHardTask` enums can be found [here](#tasks).
+- [Optional] `n_shots`: the number of "shots" to use for few-shot learning or chain-of-thought (CoT) learning. This is **set to 3 by default** and cannot exceed this number.
+- [Optional] `enable_cot`: a boolean that determines if CoT prompting is used for evaluation if `true`, and if few-shot learning if `false`. This is set to `true` by default.
+
+:::info
+**Chain-of-thought (CoT) prompting** enhances reasoning by utilizing **intermediate steps**. Combined with few-shot prompting, it improves performance on tasks needing in-depth reasoning. You can learn more about CoT [here](https://arxiv.org/abs/2201.11903).
```

Comment:
Can have half a more sentence on CoT, also few-shot was mentioned here and ^above but didn't really explain the distinction between CoTs (especially in the `n_shots` section)